pkgs/misc/vim-plugins/overrides.nix

```diff
@@ -125,6 +126,78 @@ self: super: {
     '';
   });
 
+  coc-css = buildVimPluginFrom2Nix {
+    pname = "coc-css";
+    version = nodePackages.coc-css.version;
```

Comment:
apparently, this cannot be seen:
```
attribute 'coc-css' missing ... pkgs/misc/vim-plugins/overrides.nix:131:15
```

https://gist.github.com/GrahamcOfBorg/69f8e0bd962f1ce6fc88277a0d6fecce